python/test/test_units.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+
+from dataset import *
+import numpy as np
+
+class TestUnits(unittest.TestCase):
+    def test_create_unit(self):
+        u = units.angstrom
+        self.assertEqual(repr(u), "AA")
```

Comment:
Is it possible to get an actual Angstrom symbol here (using UTF8?)

Reply:
yes